app/src/main/res/values/strings_pin.xml

```diff
@@ -8,6 +8,7 @@
     <string name="phrase3">Be attentive, there is no way to restore a \n lost PIN so far.</string>
     <string name="action_button_next">Next</string>
     <string name="pin_enter_title">Enter a pin</string>
```

Comment:
a bit late but to the party, but please make this string `Enter a PIN` to match the figma design